Sources/Model/TransferMethod/HyperwalletPayPalAccount.swift

```diff
@@ -23,7 +23,7 @@ public final class HyperwalletPayPalAccount: HyperwalletTransferMethod {
     override private init(data: [String: AnyCodable]) {
         super.init(data: data)
     }
-
+    /// Initialization of HyperwalletPayPalAccount
```

Comment:
update to - /// The required initializer

Reply:
Updated